experimental/sdk/server-proxies/src/InprocessServerProxy.ts

```diff
@@ -28,9 +28,13 @@ export interface WebPubSubServiceCaller {
     removeConnection(connectionId: string): Promise<void>;
   };
   invoke(message: string, body: (data: Uint8Array|undefined, end: boolean) => void, options?: { filter: string; contentType: string }): Promise<void>;
+  getClientAccessTokenUrl(options?: unknown): Promise<string>;
 }
 
 export class InprocessServerProxy implements WebPubSubServiceCaller {
+  public getClientAccessTokenUrl(options?: unknown): Promise<string> {
```

Comment:
no need to have this? getClientAccessTokenUrl could always call @azure/web-pubsub

Reply:
removed